worker/src/queues/notify/workers/errors.ts

```diff
@@ -11,3 +11,7 @@ export const NOTIFY_SEND_ERRORS = {
   AGGREGATE: "NOTIFY_SEND_AGGREGATE_ERROR",
   UNKNOWN: "NOTIFY_SEND_UNKNOWN_ERROR",
 };
+
+export const NOTIFY_FAILURE_ERRORS = {
+  REQUEST: "NOTIFY_FAILURE_CHECK_REQUEST_ERROR",
```

Comment:
Could you add documentation for this error code and `NOTIFY_EMAIL_FAILURE_TO_SEND` in  TROUBLESHOOTING.md please?